.ruby-version

```diff
@@ -0,0 +1,2 @@
+jruby-1.7.24
```

Comment:
Yes.  It helps make a consistent development environment and won't harm production.